content/integrations/faq/why-is-there-a-container-in-each-kubernetes-pod-with-0-cpu-and-minimal-disk-ram.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Why is there a container in each Kubernetes pod with 0% CPU and minimal disk/ram?
+kind: faq
+customnav: integrationsnav
+---
+
+These are pause containers (docker_image:gcr.io/google_containers/pause.*) that K8s injects into every pod to keep it populated even if the "real” container is restarting/stopped. 
```

Comment:
This isn't relevant as we ignore pause containers by default

Reply:
👍